grammars/ruby.cson

```diff
@@ -60,60 +60,88 @@
 '''
 'patterns': [
   {
+    'begin': '\\bclass\\b'
```

Comment:
Do classes need to be at the start of the line per the previous `^\\s*class\\s*` regex?

Reply:
No, classes can be declared after another statement on the same line.